Source/Charts/Charts/CombinedChartView.swift

```diff
@@ -120,7 +120,7 @@ open class CombinedChartView: BarLineChartViewBase, CombinedChartDataProvider
             {
                 return nil
             }
-            return (_data as! CombinedChartData!).lineData
+            return (_data as? CombinedChartData)?.lineData
```

Comment:
shall we use `(_data as! CombinedChartData)` or `(_data as? CombinedChartData)?` ? seems `as!` is safe here though as? is safer

Reply:
I want to avoid using `!`, since in a case where `_data === nil` `!` will crash